demo/components/demo/components/trees/trees.ts

```diff
@@ -0,0 +1,9 @@
+import {Component} from 'angular2/core';
+import {NGL_DIRECTIVES} from '../../../../../dist/ng-lightning';
+
+@Component({
+    selector: 'demo-trees',
+    directives: [NGL_DIRECTIVES],
```

Comment:
I think this will not work with newer angular versions, as directives must be declared on module level.